src/ApiCall.ts

```diff
@@ -696,3 +696,21 @@ export async function cardCallBack(callBackUrl: string, token: string): Promise<
     });
   });
 }
+
+// generate barcode
+export async function generateBarcode(productId: number, params: {
+  mobile_number: number,
```

Comment:
I'm not overly familiar with typescript. There's another option that could be passed in is: `scheduled_date`. It is optional, so should that be put in here as well you think? It would be a string

Reply:
And sorry, when I say another option, I mean as part of the params, so sometimes it might be just {mobile_number: '123123123'} but can also be {mobile_number: '123123123', scheduled_date: '31122024'} if that makes sense

Reply:
We can define the types as optional, or more types are okay. It won't be affected by the function. I adjusted it to more flex.

Reply:
Too easy, sounds good 👍